pkg/osquery/table/platform_tables_darwin.go

```diff
@@ -39,11 +40,10 @@ func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, c
 		kextpolicy.TablePlugin(),
 		legacyexec.TablePlugin(),
 		dataflattentable.TablePlugin(client, logger, dataflattentable.PlistType),
-		dataflattentable.TablePluginExec(client, logger,
-			"kolide_pwpolicy", dataflattentable.PlistType, []string{"/usr/bin/pwpolicy", "getaccountpolicies"}),
```

Comment:
I was thinking about whether I could write a generic way to pass args through this `exec` pattern, and alleviate the need to write full tables for each.